.github/workflows/full-integration-tests.yaml

```diff
@@ -0,0 +1,13 @@
+name: Integration Tests - Full Test Suite
+run-name: ${{ github.actor }} is running Full Integration Test Suite
+on:
+  workflow_dispatch: {}
+
+jobs:
+  call-workflow:
+    name: calling full integration test suite
+    uses: vpodorozh/github-actions/.github/workflows/full-integration-tests.yaml@full-integration-tests-execution-workflow
```

Comment:
I think this now needs to be mage-os/github-actions/.github/workflows/full-integration-tests.yaml@main

Reply:
Hej @Vinai - just updated it!